docs/guides/directing-traffic-to-your-app.md

```diff
@@ -0,0 +1,375 @@
+      Status:                True
+      Type:                  Accepted
+      Last Transition Time:  2023-08-15T20:57:21Z
+      Message:               Listener is programmed
+      Observed Generation:   1
+      Reason:                Programmed
+      Status:                True
+      Type:                  Programmed
+      Last Transition Time:  2023-08-15T20:57:21Z
+      Message:               All references are resolved
+      Observed Generation:   1
+      Reason:                ResolvedRefs
+      Status:                True
+      Type:                  ResolvedRefs
+      Last Transition Time:  2023-08-15T20:57:21Z
+      Message:               No conflicts
+      Observed Generation:   1
+      Reason:                NoConflicts
+      Status:                False
+      Type:                  Conflicted
+    Name:                    http
+  ```
+
+  Check that the conditions match and that the attached routes for the `http` listener equals 1. If it is 0, there may
+  be an issue with the HTTPRoute.
+
+- Check the status of the HTTPRoute:
+
+  ```shell
+  kubectl describe httproute coffee
+  ```
+
+  The HTTPRoute status should look similar to this:
+
+  ```text
+  Status:
+    Parents:
+      Conditions:
+        Last Transition Time:  2023-08-15T20:57:21Z
+        Message:               The route is accepted
+        Observed Generation:   1
+        Reason:                Accepted
+        Status:                True
+        Type:                  Accepted
+        Last Transition Time:  2023-08-15T20:57:21Z
+        Message:               All references are resolved
+        Observed Generation:   1
+        Reason:                ResolvedRefs
+        Status:                True
+        Type:                  ResolvedRefs
+      Controller Name:         gateway.nginx.org/nginx-gateway-controller
+      Parent Ref:
+        Group:      gateway.networking.k8s.io
+        Kind:       Gateway
+        Name:       cafe
+        Namespace:  default
+  ```
+
+  Check for any error messages in the conditions.
+
+- Check the generated nginx config:
+
+  ```shell
+  kubectl exec -it -n nginx-gateway <nginx gateway Pod> -c nginx -- nginx -T
+  ```
+
+  The config should contain the following config blocks:
+
+  ```nginx configuration
+  upstream default_coffee_80 {
+    random two least_conn;
+    zone default_coffee_80 512k;
+
+    server 10.12.0.18:8080; # these should be the Pod IPs of the coffee Pods
+    server 10.12.0.19:8080;
+  }
+  ...
+
+  server {
+    listen 80;
+
+    server_name cafe.example.com;
+
+    location / {
+
+        proxy_set_header Host $gw_api_compliant_host;
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection $connection_upgrade;
+        proxy_pass http://default_coffee_80$request_uri;
+    }
+  }
+  ```
+
+If your issue persists, open an [issue](https://github.com/nginxinc/nginx-kubernetes-gateway/issues/new/choose) in the
```

Comment:
perhaps we can leverage this section here https://github.com/nginxinc/nginx-kubernetes-gateway#contacts ?

Is opening an issue the only course of action?

Reply:
I second this. If we ever change our process, we can make a single change on our readme. Could say something like, "If your issue persists, contact us and we can take a look." With "contact us" being the hyperlink above.